hw1.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pandas as pd
+import random
+import itertools
+import librosa
+import IPython.display as ipd
+import matplotlib.pyplot as plt
+import soundfile as sf
+import os
+import argparse
+
+sound_path = './bg_noise/'
+
+def sample(path):
+    files = []
+    for dirpath, __, filenames in os.walk(path):
+        for file in filenames:
+            files.append(dirpath + '/' + file)
+    return np.random.choice(files, 1)[0]
+
+def circle(data, N):
+    n = len(data)
+    if n >= N:
+        return data[:N]
+    return np.pad(data, (0, N - n), 'wrap')
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Parameters.')
+    parser.add_argument('input', type=str, help='path to directory with input sounds')
+    parser.add_argument('noise', type=str, help='path to directory with noises')
+    parser.add_argument('power', type=float, help="power of noise, [0.01 : 0.2]")
```

Comment:
Giving such restrictions to powers seems extremely unintuitive
Consider expecting argument from 0 to 1 and normalizing the power internally